.github/CODEOWNERS

```diff
@@ -8,3 +8,4 @@
 /wiz-kubernetes-connector/ @wiz-sec/wiz-devops @wiz-sec/Wiz-Charts-Approvers circleci@wiz.io
 /.github/ @wiz-sec/wiz-devops
 /.circleci/ @wiz-sec/wiz-devops
+/.circleci/tests/ @yarinm
```

Comment:
Please change ownership here to `@wiz-sec/Wiz-Charts-Approvers`

And add an entry for `wiz-outpost-lite` with my user. I think we  can delete the other codeowner file if you do that